sites/intellij-guide/contents/tips/find-usages/index.md

```diff
@@ -0,0 +1,21 @@
+---
+date: 2020-10-05
+title: 'Find Usages'
```

Comment:
As a small note, these strings (here, and subtitle) don't need single or double quotes in YAML.